packages/core/src/messagingProtocols/proteus/ProteusService/CryptoClient/CoreCryptoWrapper/CoreCryptoWrapper.ts

```diff
@@ -154,8 +152,8 @@ export class CoreCryptoWrapper implements CryptoClient {
     return this.prekeyTracker.consumePrekey();
   }
 
-  async newPrekey(id: number) {
-    const key = await this.coreCrypto.proteusNewPrekey(id);
+  async newPrekey() {
+    const [id, key] = await this.coreCrypto.proteusNewPrekeyAuto();
```

Comment:
⚠️  This will need a coreCrypto update https://github.com/wireapp/core-crypto/blob/develop/CHANGELOG.md#070---2023-03-